js-api-doc/importer.d.ts

```diff
@@ -309,32 +309,120 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
 }
 
 /**
- * An opaque branded type, representing the built-in Node Package Importer. This
- * is used for type checking. Users should use the exposed `nodePackageImporter`
- * and not instantiate their own.
+ * An opaque branded type, representing the built-in Node.js Package Importer.
+ * This is used for type checking. Users should use the exposed
+ * {@link nodePackageImporter} and not instantiate their own.
  *
  * @category Importer
  */
 export type NodePackageImporter = {
   _NodePackageImporterBrand: unknown;
 };
+
 /**
- * The built-in Node Package Importer, which is a Package Importer that resolves
- * using the standards and conventions of the Node ecosystem. It enables a
- * `pkg:` URL scheme for usage with `@use` that directs an implementation to
- * resolve a URL within a dependency.
+ * The built-in Node.js package importer. This loads pkg: URLs from node_modules
+ * according to the standard Node.js resolution algorithm.
  *
- * A Node Package Importer is exposed as a constant that can be added to the
+ * A Node.js package importer is exposed as a constant that can be added to the
  * `importers` option.
  *
  * @example
- *
  *```js
  *const sass = require('sass');
- *sass.compile('style.scss', {
+ *sass.compileString('@use "pkg:@angular/material', {
  *  importers: [sass.nodePackageImporter]
  *});
  *```
+ *
+ * Package authors can control what is exposed to their users through their
```

Comment:
The way this is currently structured, I believe this text is considered part of the `@example` block created by the `@example` tag above. That is, I think an `@example` covers everything beneath it until the next tag, rather than just being a way of annotating a specific block of code. Given that, I think you should move the example usage to the bottom of the documentation and remove the `@example` tag from other code blocks.

Reply:
Ok, I updated this in 62f28c77578daf85d6b0e5b7bbc500f7df3212fe. I don't love that it puts the code for package users after the examples for package authors- do you have any recommended alternatives? Perhaps putting the package user example and info on the const `nodePackageImporter` and the package author info on the type `NodePackageImporter`?

Reply:
Maybe a better solution is to forego the `@example` tag altogether and just use Markdown headers to discuss different subjects. That way you can have a usage example first with just a normal code block, then `## Writing Sass packages` afterwards.

Reply:
Updated in d78c4a502235ebea2c9b3aa49e58bd0cbe28b824